resend.yaml

```diff
@@ -2127,499 +3215,580 @@ components:
+              created_at:
+                type: string
+                format: date-time
+                description: Timestamp indicating when the property was created.
+                example: '2023-10-06T22:59:55.977Z'
+              updated_at:
+                type: string
+                format: date-time
+                description: Timestamp indicating when the property was last updated.
+                example: '2023-10-06T22:59:55.977Z'
+    UpdateContactPropertyOptions:
       type: object
       properties:
-        name:
-          type: string
-          description: The name of the template.
-        alias:
-          type: string
-          description: The alias of the template.
-        from:
-          type: string
-          description: Sender email address. To include a friendly name, use the format "Your Name <sender@domain.com>".
-        subject:
-          type: string
-          description: Email subject.
-        reply_to:
-          type: array
-          items:
-            type: string
-          description: Reply-to email addresses.
-        html:
-          type: string
-          description: The HTML version of the template.
-        text:
-          type: string
-          description: The plain text version of the template.
-        variables:
-          type: array
-          items:
-            $ref: '#/components/schemas/TemplateVariableInput'
-    UpdateTemplateResponseSuccess:
+        fallback_value:
+          oneOf:
+            - type: string
+            - type: number
+          description: The default value to use when the property is not set for a contact. Must match the type of the property.
+    UpdateContactPropertyResponseSuccess:
       type: object
       properties:
         id:
           type: string
-          description: The ID of the template.
-        object: 
+          description: The ID of the contact property.
+          example: 'b6d24b8e-af0b-4c3c-be0c-359bbd97381e'
+        object:
           type: string
           description: The object type of the response.
-          example: template
-    RemoveTemplateResponseSuccess:
+          example: contact_property
+    RemoveContactPropertyResponseSuccess:
       type: object
       properties:
+        id:
+          type: string
+          description: The ID of the contact property.
+          example: 'b6d24b8e-af0b-4c3c-be0c-359bbd97381e'
         object:
           type: string
           description: Type of the response object.
-          example: template
-        id:
-          type: string
-          description: The ID of the template.
+          example: contact_property
         deleted:
           type: boolean
-          description: Indicates whether the template was successfully deleted.
+          description: Indicates whether the contact property was successfully deleted.
           example: true
-    PublishTemplateResponseSuccess:
+    AddContactToSegmentResponseSuccess:
```

Comment:
Not necessarily related to this API spec, but aren't we missing an `object` in this return? 

https://resend.com/docs/api-reference/contacts/add-contact-to-segment